### ISSUE TITLE
Webhook is not trigger on both branches on push

### ISSUE THREAD

**Issue by MikeVL** [gogits/gogs#3337]
- Gogs version (or commit ref): 0.9.48.0722
- Git version: 2.7.4
- Operating system:
- Database:
  - [ ] PostgreSQL
  - [*] MySQL
  - [ ] SQLite
- Can you reproduce the bug at http://try.gogs.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [ ] Not relevant
- Log gist:
## Description

After `git merge --no-ff develop` to master branch, i push both branches to my git repository.
Webhook trigger only on develop branch, master is not trigger.
My default branch is develop
...

**Comment by Unknwon** [comment]
>Can you reproduce the bug at http://try.gogs.io:
 Yes (provide example URL)
 No
 Not relevant

**Comment by MikeVL** [comment]
At http://try.gogs.io webhook triggered twice. It's works fine.

**Comment by Unknwon** [comment]
Can you see your push actions show in your dashboard news feed for both two branches?

**Comment by MikeVL** [comment]
Only one branch
<img width="643" alt="2016-08-03 13 53 37" src="https://cloud.githubusercontent.com/assets/118285/17363285/8307a054-598a-11e6-9370-cde839b4e7e4.png">


<img width="724" alt="2016-08-03 13 54 03" src="https://cloud.githubusercontent.com/assets/118285/17363228/31747712-598a-11e6-9052-f9587023a5c8.png">

**Comment by Unknwon** [comment]
I see you use SSH push on your Gogs, are you using SSH too in try.gogs.io?

**Comment by MikeVL** [comment]
Ho, to push to try.gogs.io i use https

**Comment by Unknwon** [comment]
OK, please test with HTTP on your instance as well.

**Comment by MikeVL** [comment]
When push via ssh to try.gogs.io is not working.
```
Counting objects: 4, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (2/2), done.
Writing objects: 100% (4/4), 439 bytes | 0 bytes/s, done.
Total 4 (delta 0), reused 0 (delta 0)
To git@try.gogs.io:Elec/webhook-test.git
   1574212..9a5a5bd  develop -> develop
   4e975e9..305944a  master -> master
Everything up-to-date
```
![2016-08-03 20 29 14](https://cloud.githubusercontent.com/assets/118285/17375287/5c2a76c8-59c1-11e6-9139-a807000e5f45.png)

**Comment by Unknwon** [comment]
Thanks, looks like push via SSH does not detect multiple branches.

**Comment by lstahlman** [comment]
That last dashboard screenshot from @MikeVL doesn't look right.  In the second item in the feed, it says "MikeVL pushed to **master** at Elec/webhook-test", but one of the two commits referenced was pushed to **develop**.  There could be an additional bug here on the HTTP side.

**Comment by Unknwon** [comment]
This issue is fixed, you can test on `release/0.11` or demo site.
